index.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Responsive website</title>
+    <link rel="stylesheet" href="mainStyle.css">
+    <link rel="stylesheet" media="(min-width: 980px)" href="psd1024px.css">
+    <link rel="stylesheet" media="(min-width: 640px) and (max-width: 980px)" href="psd640_980px.css">
+    <link rel="stylesheet" media="(min-width: 320px) and (max-width: 640px)" href="psd320_640px.css">
+    <link rel="stylesheet" media="(min-width: 1px) and (max-width: 320px)" href="psd0_320px.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.0/css/all.css" 
+    integrity="sha384-lZN37f5QGtY3VHgisS14W3ExzMWZxybE1SJSEsQp9S+oqd12jhcu+A56Ebc1zFSJ" crossorigin="anonymous">
+</head>
+<body>
+    <nav class="navigation">
+
+            <div id="NavLabel">
```

Comment:
Looks like complex structure, it's better to use h3 Navigation for example

Reply:
fixed